src/DataFrames.jl

```diff
@@ -51,6 +52,7 @@ export AbstractDataFrame,
        nrow,
        order,
        outerjoin,
+       PrettyTables,
```

Comment:
I understand we export just the name `PrettyTables`. I think it is OK.

Reply:
Yes, we are exporting just the symbol `PrettyTables`. We can remove this, but I think it was requested by @nalimilan IIRC.

Reply:
I don't really remember TBH. Can you remind me why it's useful? (Maybe it's changed since we first discuss it?)

Reply:
This was such a huge discussion that I have also no idea why it was thought to be useful, sorry 😅 In fact, this is only useful if the user wants to extend printing. For example, let's say the user wants to change the format of vertical lines:

```julia
julia> show(df, tf = PrettyTables.tf_ascii_dots)
1×2 DataFrame
 Row : a      b
     : Int64  Int64
.....:..............
   1 : 1      2
```

Or, if the user wants to highlight cells, then it can use `PrettyTables.Highlighter` to create custom highlighters, or use the pre-defined ones such as `PrettyTables.hl_lt`. If we do not export this symbol, then the user needs `using PrettyTables`.

I have no opinion about it, what do you prefer?

Reply:
I think it is ok to keep it. name `PrettyTables` for sure will not conflict with anything normal.